R/trace_calls.R

```diff
@@ -43,6 +43,12 @@ trace_calls <- function (x, parent_functions = NULL, parent_ref = NULL) {
   }
   else if (is.call(x)) {
     src_ref <- attr(x, "srcref") %||% impute_srcref(x, parent_ref)
+    for (i in seq_along(src_ref)) {
```

Comment:
here for example, there might be a comment such as:

`impute_srcref` also identifies branches for R control structures (for, if, switch and while). This information is only need here to include the given srcref into the global .branches environment.